### ISSUE TITLE
Tabular data?

### ISSUE THREAD

**Issue by Bartolomeus-649** [migueldeicaza/gui.cs#255]
Does gui.cs handle tabular data?
* Auto sizing columns?
* line breaks within columns
* truncate column content?
* scroll the table if needed
 and so on?

Anything like this or something similar?

**Comment by migueldeicaza** [comment]
No, but that would be a cool control - a CSV control for ListViews.

```
public class CsvListView : ListView {
    public CsvListView () {}
    public string Separator { get; set; }
    public string Quotes { get; set; }
}
```

**Comment by tig** [comment]
Closing as this is materially addressed with #1015
